benchmark.sh

```diff
@@ -1,32 +1,18 @@
 #!/bin/bash
 
-# Run this script to benchmark the NBD server using nbd-client and hdparm.
+# Run this script to benchmark the NBD server using qemu-img.
 
 set -eux
 
 dd if=/dev/zero of=/tmp/test bs=1M count=100
-_build/default/cli/main.exe serve --no-tls /tmp/test &
+_build/default/cli/main.exe serve --exportname test --no-tls /tmp/test &
 SERVER_PROCESS=$!
-echo $SERVER_PROCESS
+# Wait for the server to start the main loop
+sleep 0.1
```

Comment:
Isn't there some way the server could notify that it has started?

Reply:
I think there might be, maybe we could wait until something is listening on the 10809 port, but I wanted to keep things simple in the tests.